svc/balance/service.go

```diff
@@ -0,0 +1,79 @@
+package balance
+
+import (
+	"context"
+	"log"
+
+	"github.com/SatorNetwork/sator-api/svc/wallet"
+	"github.com/google/uuid"
+)
+
+type (
+	// Service struct
+	Service struct {
+		wallets walletsServiceClient
```

Comment:
I'm not sure if `service` is required for better understanding, wallets/rewardsClient is okay as it is I believe